### ISSUE TITLE
[SF][0.23.2]DynamicPageTitle]Overflow button of DynamicPageTitle - actions toolbar should have popup a11y attribute

### ISSUE THREAD

**Issue by ksblklu** [SAP/ui5-webcomponents-react#3001]
Since this is a button that opens a pop over menu, aria-haspopup="menu" should be present on the "button" element so that the "button menu" announcement is made by screen reader.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/27538066/170927617-26d8a276-209e-40cb-8130-b2d498d8957e.png">

**Comment by Lukas742** [comment]
Hi @ksblklu 

the aria attribute is already implemented and is also read out by the nvda screen reader. If you're still having problems with it please let us know by providing a reproducible example or a more expressive description of the problem. 

```html
<button type="button" class="ui5-button-root" data-sap-focus-ref="" part="button" aria-pressed="false" tabindex="0" aria-label="Mehr anzeigen" title="Mehr anzeigen" aria-expanded="false" aria-haspopup="menu">
...
</button>
```
